### PR TITLE
similarity: threshold transformer

### PR DESCRIPTION
**Pull request by MSusik**
Signed-off-by: Mateusz Susik <mateuszsusik@gmail.com>

### CODE REVIEW

**Comment by MSusik** [comment]
Review from email by @natsheh 

> After a quick check, all seems to be fine.

Merging.

File: beard/similarity/pairs.py [hunk 1]
```diff
@@ -330,6 +330,51 @@ def transform(self, X):
         return Xt
 
 
+class Thresholder(TransformerMixin):
+    """Element-wise multiplication on paired data."""
```
Comment (natsheh) [inline beard/similarity/pairs.py]: Need to change this doc

File: examples/applications/author-disambiguation/distance.py [hunk 2]
```diff
@@ -263,20 +264,21 @@ def _build_distance_estimator(X, y, verbose=0, ethnicity_estimator=None,
                 ("classifier", EstimatorTransformer(ethnicity_estimator)),
             ]), groupby=group_by_signature)),
             ("sigmoid", FuncTransformer(func=expit)),
-            ("combiner", ElementMultiplication())
+            ("combiner", ElementMultiplication()),
+            ("thresholder", Thresholder(0.25))
```
Comment (natsheh) [inline examples/applications/author-disambiguation/distance.py]: better to pass the threshold value as an argument

Comment (MSusik) [inline examples/applications/author-disambiguation/distance.py]: @natsheh Do you mean that the threshold should be passed from the command line?

**Comment by kaplun** [comment]
Hi! What are you working on? What impact is it going to have on our usage at INSPIRE?

File: examples/applications/author-disambiguation/distance.py (hunk 2 above)
Comment (natsheh) [inline examples/applications/author-disambiguation/distance.py]: yes, and the 0.25 could be the default

**Comment by MSusik** [comment]
@kaplun We're adjusting one thing for our publication. Specifically, we're changing the way how the ethnicity features are used. These changes will be probably reverted, and if not, they won't have much impact at INSPIRE (we will leave them if they improve the accuracy). Anyway, I believe you won't be using ethnicity features for the disambiguation at INSPIRE, so you won't notice any difference.

Sorry for some glitches, like no commit log, but we're a but in a hurry (you know, deadlines :smile:)

**Pull request merged**